src/devices/amina.ts

```diff
@@ -77,25 +85,21 @@ const fzLocal = {
             const result: KeyValue = {};
 
             if (msg.data.alarms !== undefined) {
-                result.alarms = [];
+                const activeAlarms = [];
                 result.alarm_active = false;
 
                 for (let i = 0; i < 16; i++) {
                     if ((msg.data['alarms'] >> i) & 0x01) {
-                        let alarm = aminaAlarms[i];
-                        if (alarm === undefined) {
-                            alarm = `Unknown Alarm bit #${i}`;
-                        }
-
-                        (result.alarms as string[]).push(alarm);
+                        activeAlarms.push(aminaAlarmsEnum.values[i + 1]);
                         result.alarm_active = true;
                     }
                 }
 
                 if (result.alarm_active === false) {
-                    result.alarms = 'No Alarm';
+                    activeAlarms.push(aminaAlarmsEnum.values[0]);
```

Comment:
What is the reasoning for publishing the `no_alarm`? I would have expected an empty list if there were no alarms. This seems to just setup everyone for having to filter out the `no_alarm` string.

Reply:
Fair point I guess, will remove it ...

Reply:
Fixed in https://github.com/Koenkk/zigbee-herdsman-converters/pull/8191/commits/f23e32bf96e13913a368ca1db34d75dd92612350